### ISSUE TITLE
IE11+NVDA doesn't read aria-setsize if active element has aria-describedby property

### ISSUE THREAD

**Issue by Leuvarden** [nvaccess/nvda#11853]
### Steps to reproduce:

1. Open https://jsbin.com/kixenulaqu/1/edit?html,output in Internet Explorer with NVDA on
2. Navigate through the radio buttons via up/down arrows

### Actual behavior:

NVDA reads "1 of 8", "2 of 8" for four first buttons without aria-describedby, but doesn't read indexes for the four last buttons
![image](https://user-images.githubusercontent.com/16884735/99911198-be8d1180-2d03-11eb-962d-d575bd8e1ac0.png)

### Expected behavior:

NVDA should read indexes for all radiobuttons

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:

NVDA 2020.3

#### Windows version:

Microsot Windows 10 Enterprise
10.0.18363 Build 18363

#### Name and version of other software in use when reproducing the issue:

Internet Explorer 11.1139.18362.0
Update Versions 11.0.210

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No

#### If addons are disabled, is your problem still occuring?

Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?

No

**Comment by michaelDCurran** [comment]
Technical:
We seem to rely on the fact that Internet Explorer exposes the group index info (E.g. 4 of 8) via IAccessible::accDescription, yet when a real description is provided by ARIA, NVDA specifically overrides the NVDAObject's description property to expose the ARIA description, and not what was in accDescription.
Assuming we can ensure that MSHTML NVDAObject's description property can correctly handle all possible ways an author can provide a description using the DOM, then we should no longer rely on IAccessible::accDescription at all. And to correctly expose group information, manually etect aria posInSet and setSize from the DOM and expose that via the NVDAObject's positionInfo property. 
As this is Internet Explorer, this is extremely low priority for us, but we'd accept a pr, as long as all the above was carefully considered.

**Comment by Adriani90** [comment]
Support for IE ended completely in 2022 and the browser is removed automatically from machines that still have it after an Edge update in February 2023. I don't expect any accessibility updates for the IE mode in Edge, so i am closing this issue as abandoned.
